### PR TITLE
Create basic_data.php

### PR DESCRIPTION
**Pull request by TheRealEdDawson**
## Checklist

- [X] Documentation

### CODE REVIEW

**Comment by TheRealEdDawson** [comment]
This is a small update to the bundled examples and HTML pages to support the new Data API quickstart guide: https://learnositysupport.zendesk.com/knowledge/articles/360000758637/en-us?brand_id=647421&return_to=/hc/en-us/articles/360000758637

File: docs/quickstart/analytics/basic_data.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+/* Copyright (c) 2023 Learnosity, MIT License
+ * Basic example code for pulling information from the Learnosity cloud using Data API. */
+
+// Setup to load the necessary classes from the example directory, & set up variables for access.
+require_once __DIR__ . '/../../../bootstrap.php'; 
+$config = require_once __DIR__ . '/../config.php'; // Load security keys from config.php, for Learnosity's public demos account.
+use LearnositySdk\Request\DataApi; // Load core Data API library.
+$itembank_uri = 'https://data.learnosity.com/v1/sessions/responses'; // Choosing the Data API endpoint for sessions/responses.
```
Comment (kent-nguyen-lrn) [inline docs/quickstart/analytics/basic_data.php:9]: Should we use an LTS version here instead of `v1`?

**Review by kent-nguyen-lrn — APPROVED** [review]
Looks good, just left some suggestions

File: docs/quickstart/analytics/basic_data.php (hunk 1 above)
Comment (TheRealEdDawson) [reply docs/quickstart/analytics/basic_data.php]: Yes! Great pickup. Updated.

File: docs/quickstart/index.html [hunk 2]
```diff
@@ -38,8 +38,12 @@ <h3><i class="li-reportsapi hero-icon"></i> Analytics examples</h3>
...
 			</li>
 			<li>
-				<a href="analytics/init_data.php">Data API access</a>
-				<p>Pull some raw information from the system. <a href="https://github.com/Learnosity/learnosity-sdk-php/blob/master/REFERENCE.md">Learn more</a>.</p>
+				<a href="analytics/basic_data.php">Basic data extraction</a>
+				<p>Pull some specific data from the system with Data API.</p>
+			</li>
+			<li>
+				<a href="analytics/init_data.php">Paginated data extraction</a>
+				<p>Pull random data from the system with Data API, in paginated chunks. <a href="https://github.com/Learnosity/learnosity-sdk-php/blob/master/REFERENCE.md">Learn more</a>.</p>
```
[the remark this answers is not in the document]
Comment (TheRealEdDawson) [reply docs/quickstart/index.html]: Nice. Updated.

**Pull request merged**